VRDR.Messaging/CodingResponseMessage.cs

```diff
@@ -283,31 +241,32 @@ public List<CauseOfDeathEntityAxisEntry> CauseOfDeathEntityAxis
     /// <summary>Class for structuring a cause of death entity axis entry</summary>
     public class CauseOfDeathEntityAxisEntry
     {
-        /// <summary>A line of the original cause of death on the death certificate</summary>
-        public readonly string DeathCertificateText;
-
-        /// <summary>The identifier of the corresponding cause of death condition entry in the VRDR FHIR document</summary>
-        public readonly string CauseOfDeathConditionId;
+        /// <summary>The line identifier of the corresponding cause of death in the death certificate</summary>
+        /// <list type="number">
+        /// <item>Part I. Line a</item>
+        /// <item>Part I. Line b</item>
+        /// <item>Part I. Line c</item>
+        /// <item>Part I. Line d</item>
+        /// <item>Part I. Line e</item>
+        /// <item>Part II</item>
+        /// </list>
+        public readonly string LineNumber;
```

Comment:
Perhaps I'm misunderstanding how this documentation works, but I don't see how this &lt;list /&gt; applies to LineNumber

Reply:
Each item is a value of the line number parameter. E.g. if LineNumber is 1, that represents Part I. Line a (as per the TRANSAX format). I did consider adding an enum to make this even clearer?